Resources/Prototypes/Catalog/thief_toolbox_sets.yml

```diff
@@ -83,21 +83,22 @@
   id: CommunicatorSet
   name: thief-backpack-category-communicator-name
   description: thief-backpack-category-communicator-description
-  sprite:
+  sprite: 
     sprite: Objects/Tools/spy_device.rsi
     state: icon
   content:
   - EncryptionKeyStationMaster
   - CyberPen
+  - SpyCrewMonitor
```

Comment:
This was also removed in Resources/migration.yml and needs to be removed here or returned